src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -38,11 +44,12 @@ public void preInit(FMLPreInitializationEvent event) {
 
         configPath = event.getModConfigurationDirectory() + "/RTG/";
         Mods.syncAllConfigs();
+        
+        // This must come before the event manager is registered.
+        this.registerStructures();
```

Comment:
1. Why move the registrations to a method?
2. What specific event(s) does the registration have to happen before? and why?

Being that the world gen specific events don't get registered until the server is going to start up, we can probably register these at anytime during the pre-init or init process. (I had originally moved the registrations in 1.7.10 to post-init and I don't recall noticing any side effects.)